runner/README.md

```diff
@@ -15,9 +15,13 @@ If the user elects to regenerate, or if there is no set to reuse, then the relev
 
 ## Scheduling
 
-Once the `TestExample` objects are collected, the scheduler creates groups of tests. A test group is a collection of non-interfering tests, for example the tests `names` and `colours` do not interfere, as the names and favourite colours of a person are not mutually exclusive. However, `names` and `name_list` will interfere with each other, as their test examples have different names that they will supply to the agent. By default, tests interfere with themselves, so a typical group will take one `TestExample` object from each dataset.
+Once the `TestExample` objects are collected, the system will perform a check for whether it can resume the test. If not, then the tests will start from scratch. If the tests are to be resumed, the master log is read, a list of in progress tests are collected and each of those tests
+are "fast forwarded" to last moment in the script that they provided to the agent. Once this is done, the running of the tests resume.
```

Comment:
This should go in a single line.

Reply:
Nah, its latex rules - two newlines for a line break: https://github.com/GoodAI/goodai-ltm-benchmark/tree/update-runner-readme/runner

Reply:
It might also be good to provide a more user-friendly description of what happens e.g. omit the details about the master log and just say that the test resumes from where it last was.